charts/prometheus-elasticsearch-exporter/README.md

```diff
@@ -0,0 +1,86 @@
+# Elasticsearch Exporter
+
+Prometheus exporter for various metrics about ElasticSearch, written in Go.
+
+Learn more: <https://github.com/justwatchcom/elasticsearch_exporter>
+
+This chart creates an Elasticsearch-Exporter deployment on a [Kubernetes](http://kubernetes.io)
+cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.8+ with Beta APIs enabled
+
+## Get Repo Info
+
+```console
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm repo update
+```
+
+_See [helm repo](https://helm.sh/docs/helm/helm_repo/) for command documentation._
+
+## Install Chart
+
+```console
+# Helm 3
+$ helm install [RELEASE_NAME] prometheus-community/prometheus-stackdriver-exporter --set stackdriver.projectId=google-project-name
+
+# Helm 2
+$ helm install --name [RELEASE_NAME] prometheus-community/prometheus-stackdriver-exporter --set stackdriver.projectId=google-project-name
+```
+
+The command deploys Stackdriver-Exporter on the Kubernetes cluster using the default configuration.
+
+_See [configuration](#configuration) below._
+
+_See [helm install](https://helm.sh/docs/helm/helm_install/) for command documentation._
+
+## Uninstall Chart
+
+```console
+# Helm 3
+$ helm uninstall [RELEASE_NAME]
+
+# Helm 2
+# helm delete --purge [RELEASE_NAME]
+```
+
+This removes all the Kubernetes components associated with the chart and deletes the release.
+
+_See [helm uninstall](https://helm.sh/docs/helm/helm_uninstall/) for command documentation._
+
+## Upgrading Chart
+
+```console
+# Helm 3 or 2
+$ helm upgrade [RELEASE_NAME] [CHART] --install
+```
+
+_See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documentation._
+
+### To 3.0.0
+
+`prometheusRule.rules` are now processed as Helm template, allowing to set variables in them.
+This means that if a rule contains a {{ $value }}, Helm will try replacing it and probably fail.
+
+You now need to escape the rules (see `values.yaml`) for examples.
+
+### To 2.0.0
+
+Some Kubernetes apis used from 1.x have been deprecated. You need to update your cluster to Kubernetes 1.10+ to support new definitions used in 2.x.
```

Comment:
Are these applicable to elasticsearch-exporter or stackdriver-exporter? Just checking due to the other typos, and can't immediately tell since I'm not familiar with these charts.

Reply:
This is ok:
https://github.com/helm/charts/tree/master/stable/elasticsearch-exporter#to-200
https://github.com/helm/charts/tree/master/stable/elasticsearch-exporter#to-300